src/common/utils/membershipHelpers.js

```diff
@@ -1,7 +1,12 @@
 // FPCC
-import { atLeastMember } from 'common/constants/roles'
+import { atLeastMember, atLeastEditor } from 'common/constants/roles'
 
 export const isMember = ({ user, sitename }) => {
   const userSiteRole = user?.roles?.[sitename] || ''
   return userSiteRole.match(atLeastMember)
 }
+
+export const isAtLeastEditor = ({ user, sitename }) => {
```

Comment:
If we want, these methods can also be combined into one method, e.g. 
```
export const isMinRole({ user, sitename, role }) => {
  const userSiteRole = user?.roles?.[sitename] || ''
  return userSiteRole.match(role)
}
```
and then it can be used for all roles e.g. atLeastMember, atLeastEditor and so on..

Reply:
This is a great idea. I can see us needing other isAtLeast{role} methods in the future and this would cover us for those as well. I'll leave it up to you if you want to make that refactor now or not.